Source/Xamarin/Prism.Forms/Navigation/PageNavigationService.cs

```diff
@@ -375,6 +376,18 @@ protected virtual Page CreatePageFromSegment(string segment)
             }
         }
 
+        protected void ApplyPageBehaviors(Page page)
```

Comment:
Well if we wanted to get crazy, we could implement an `IPageBehaviorFactory` and provide a `ConfigureDefaultPageBehaviors` method in the application to make it easier to apply behaviors to a page type.  This would be almost exactly how WPF does it.  Thoughts?